routes/spec.js

```diff
@@ -442,6 +442,406 @@ const spec = {
                             },
                           },
                         },
+                        Bedwars: {
+                          description: 'Player stats in Bedwars',
+                          type: 'object',
+                          properties: {
+                            coins: {
+                              description: 'Current coins in Bedwars',
+                              type: 'integer',
+                            },
+                          },
+                        },
+                        BuildBattle: {
+                          description: 'Player stats in Build Battle',
+                          type: 'object',
+                          properties: {
+                            coins: {
+                              description: 'Current coins in Build Battle',
+                              type: 'integer',
+                            },
+                          },
+                        },
+                        Duels: {
+                          description: 'Player stats in Duels',
+                          type: 'object',
+                          properties: {
+                            coins: {
+                              description: 'Current coins in Duels',
+                              type: 'integer',
+                            },
+                          },
+                        },
+                        GingerBread: {
```

Comment:
The API returns game names in [standard notation](https://github.com/slothpixel/hypixelconstants/blob/master/build/game_types.json), so some of these need to be changed.

Reply:
My bad, missed that when looking through. Will refactor properly alongside next commit when I finish BSG.

Reply:
Game keys should be updated to standard notation now, closing review